aim/web/ui_v2/src/components/ChartPanel/config.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+interface IChartTypeConfig {
+  [key: string]: React.LazyExoticComponent<any>;
+}
+
+export const chartTypesConfig: IChartTypeConfig = {
+  LineChart: React.lazy(() => import('../LineChart/LineChart')),
+  HighPlot: React.lazy(() => import('../HighPlot/HighPlot')),
+};
```

Comment:
I think we can delete config file, move chartTypesConfig to ChartPanel component and write IChartTypeConfig in IChartPanelProps